common/src/main/java/tachyon/network/protocol/RPCBlockReadResponse.java

```diff
@@ -128,4 +125,16 @@ public long getOffset() {
   public Status getStatus() {
     return mStatus;
   }
+
+  /**
+   * Release the underlying DataBuffer in this read response.
+   *
+   * @return true if successfully released or no release is needed; otherwise false.
+   */
+  public boolean releaseBuffer() {
```

Comment:
For releasing the DataBuffer, I think the other classes can just call "getPayloadDataBuffer()" to get the DataBuffer, and then call release() on the data buffer when it is done. This would mean that the DataBuffer interface would need a release() method.

This would also mean there should be a comment on the getPayloadDataBuffer() interface that release() must be called on it.

I just didn't think that release() or close() on a message object made as much sense as releasing a DataBuffer. What do you think?